<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="Background" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1201</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#Background{
Background-color:qlineargradient(spread:pad, x1:0.46, y1:0.977318, x2:0, y2:0.977, stop:0.494318 rgba(222, 230, 255, 255), stop:0.505 rgba(202, 203, 255, 255), stop:1 rgba(183, 208, 255, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 36pt &quot;Seagull&quot;;</string>
    </property>
    <property name="text">
     <string>Inventory Bot</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255);
font: 22pt &quot;Seagull&quot;;</string>
    </property>
    <property name="text">
     <string>Target</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;MS Shell Dlg 2&quot;;
color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>481</width>
      <height>87</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Seagull&quot;;
background:rgba(0,0,0,0)
</string>
    </property>
    <row>
     <property name="text">
      <string>Ps5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Xbox</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Amount</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Profit</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0,0,0,0);
font: 22pt &quot;Seagull&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;- Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Amazon.qrc"/>
 </resources>
 <connections/>
</ui>
